Mathlib/Dynamics/TopologicalEntropy/DynamicalEntourage.lean

```diff
@@ -38,100 +38,90 @@ these dynamical entourages.
 namespace Dynamics
 
 open Prod Set UniformSpace
-open scoped Topology Uniformity
+open scoped Rel Topology Uniformity
 
 variable {X : Type*}
 
 /-- The dynamical entourage associated to a transformation `T`, entourage `U` and time `n`
 is the set of points `(x, y)` such that `(T^[k] x, T^[k] y) ∈ U` for all `k < n`, i.e.
```

Comment:
I think if you're changing these definitions to use `Rel`, then the description should not say "set of points"